lib/webapp.js

```diff
@@ -1781,7 +1788,8 @@ WebApp.prototype = {
       for (const version of Object.keys(group.versions)) {
         const service = group.versions[version];
         const subUrl = urlBase + zLuxUrl.makeServiceSubURL(service);
-        const handle = new WebServiceHandle(subUrl, this.wsEnvironment);
+        const isProxy = (service.type === 'service' || service.type === 'external') ? service: false;
+        const handle = new WebServiceHandle(subUrl, this.wsEnvironment, isProxy);
```

Comment:
The name `isProxy` assumes that the variable is boolean, but actually the type is `object | false`. 
I think you could pass `service` into `WebServiceHandle` and check if the service type is `service or external or other` inside of the method  `WebServiceHandle.call(...)`.